github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```
@@ -542,6 +542,39 @@ presubmits:
             memory: 4Gi
         securityContext:
           privileged: true
+  - always_run: false
+    annotations:
+      fork-per-release: "true"
+    cluster: kubevirt-prow-control-plane
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-podman-in-container-enabled: "true"
+    name: pull-kubevirt-build-s390x
+    optional: true
+    skip_branches:
+    - release-\d+\.\d+
+    spec:
```

I'd suggest to not report the lane right now, to avoid causing friction.

```suggestion
    skip_report: true
    spec:
```

---

Agree, since it's not always_run and optional it should not get reported by default. I was just thinking about people accdentally triggering it and then wondering, but I guess that should not happen.